docs/reference/dapis/understand/index.md

```diff
@@ -50,47 +52,25 @@ to get the value of a dAPI.
 ## The role of Airnode
 
 Airnode is a flexible off-chain module that can support multiple protocols. Most
-noticeably are its implementation of the request-response protocol (RRP) and
-data feeds.
+noticeably is its implementation of the request-response protocol (RRP) and data
+feeds.
 
 An Airnode is owned by an API provider and integrates itself along side their
-API operations. Functionality within Airnode monitors one or more API operations
-hosted by an API provider and looks for a preset deviation of a data feed value
-(e.g., plus or minus 1%). When the deviation threshold is reached, Airnode
-pushes the new value on-chain into the beacon store. Each beacon represents a
-value from a single API operation.
+API operations. Airnode interfaces with these operations and gathers its signed
```

Comment:
I'm not sure what "integrates itself along side" means. Perhaps something like "An Airnode is owned by an API provider and is used to call API provider endpoints to fetch and sign data. Airseeker uses the signed data..."

Reply:
Yes that was a run of the mouth there. Done

Reply:
Looks like the old text meant to be replaced is still there: "An Airnode is owned by an API provider and integrates itself along side their API operations. Airnode interfaces with these operations and gathers its signed data at the request of Airseeker." Everything else looks good to me.

Reply:
@dcroote  I see the mistake. I added the new line but did not remove the old one.